XRTK.Lumin/Packages/com.xrtk.lumin/Inspectors/LuminHandControllerDataProviderProfileInspector.cs

```diff
@@ -38,14 +46,19 @@ public override void OnInspectorGUI()
 
             serializedObject.Update();
 
-            EditorGUILayout.LabelField("Oculus Hand Settings", EditorStyles.boldLabel);
+            showLuminHandTrackingSettings = EditorGUILayoutExtensions.FoldoutWithBoldLabel(showLuminHandTrackingSettings, new GUIContent("Lumin Hand Tracking Settings"), true);
+            if (showLuminHandTrackingSettings)
+            {
+                EditorGUI.indentLevel++;
 #if PLATFORM_LUMIN
             keyPointFilterLevel.intValue = (int)(MLKeyPointFilterLevel)EditorGUILayout.EnumPopup(keyPointContent, (MLKeyPointFilterLevel)keyPointFilterLevel.intValue);
             poseFilterLevel.intValue = (int)(MLPoseFilterLevel)EditorGUILayout.EnumPopup(poseFilterContent, (MLPoseFilterLevel)poseFilterLevel.intValue);
 #else
-            EditorGUILayout.PropertyField(keyPointFilterLevel);
-            EditorGUILayout.PropertyField(poseFilterLevel);
-#endif // PLATFORM_LUMIN
+                EditorGUILayout.PropertyField(keyPointFilterLevel);
+                EditorGUILayout.PropertyField(poseFilterLevel);
```

Comment:
Let's use the GUI Content we created in OnEnable

Reply:
Not sure I understand. The GUI Content created in OnEnable is being used, but only if on the PLATFORM_LUMIN.